Hardware/index.md

```diff
@@ -40,20 +40,32 @@ More information can be found on the [Verified Configurations](../projects/sel4/
 
 ### Not in the lists below?
 
-If the platform, architecture, feature that you are after is not listed below, your
-options are:
-
-* contact one of the seL4 Foundation [Endorsed Services
+If the platform, architecture, feature that you are after is not listed below,
+you have several options, listed below. It is important to note however that, as
+explained in the guidelines linked below, new ports or features will require
+compelling arguments, discussion with the technical community (including through
+Request-For-Comments), as well as testing requirements and maintenance/expertise
+commitment, to a degree depending on the nature of the contribution.
+
+* You can check the [roadmap](../projects/roadmap.html) for any planned
+  contributions, from the seL4 Foundation or larger community, such on any new
+  architecture ports, new large formal verifications, or large or fundamental
+  new features;
+
+* You can inquire the seL4 community, through one of our [communication
+  channels](https://sel4.systems/contact/) as to whether someone is developing
+  such a port or feature already, or as to the general interest in discussing
+  such such a new port or feature;
+
+* If you are in a position to develop the seL4 port or feature yourself, you
+  should follow our [guidelines for contributing kernel
+  code](../projects/sel4/kernel-contribution.html), which details the
+  implications and expectations;
+
+* You can also contact one of the seL4 Foundation [Endorsed Services
```

Comment:
I liked that the service providers were high up in the list before. Unless there is a good reason not to, we should keep them at second or first place.

Reply:
back in 1st position